LibreHardwareMonitorLib/Hardware/Cpu/IntelCpu.cs

```diff
@@ -401,6 +403,8 @@ public IntelCpu(int processorIndex, CpuId[][] cpuId, ISettings settings) : base(
                         if (!Ring0.ReadMsr(_energyStatusMsrs[i], out eax, out uint _))
                             continue;
 
+                        if (i == 2 && !Software.OperatingSystem.IsUnix)
+                            continue;
```

Comment:
?

Reply:
I wanted to remove the "GPU Power" sensor from the CPU so that there are no duplicates, bit strange maybe, will revert.

Reply:
Ah, makes sense, maybe add a comment then :)